src/js/tech/html5.js

```diff
@@ -962,6 +1079,27 @@ Html5.supportsNativeTextTracks = function() {
   return supportsTextTracks;
 };
 
+/*
+ * Check to see if native video tracks are supported by this browser/device
+ *
+ * @return {Boolean}
+ */
+Html5.supportsNativeVideoTracks = function() {
```

Comment:
I like the `supportsNativeTracks` function from #3166 but I think that the supportsNativeTracks function actually needs to be specific to each type of track. Right now in `supportsNativeTextTrack`  we check a lot of very specific things that may or may not apply to Video and Audio tracks. I could see merging the audio and video track functions but it may just end up being confusing that way as we would always have a specific text function.